docs/develop/node/archival/run-archival-node.md

```diff
@@ -0,0 +1,60 @@
+---
+id: run-archival-node
+title: Run an Archival Node
+sidebar_label: Run an Archival Node
+description: Run an Archival Node
+---
+
+Running an archival node is the same as a [validator node](/docs/develop/node/validator/running-a-node) as both types of node use the same `nearcore` release. The key difference for running an archival node is a modification to the `config.json` by changing `archive` to `true` and specifying `tracked_shards`.
+
+The `config.json` should contain the following fields. Currently, NEAR testnet and mainnet have only 1 (indexed [0]) shard and that shard is tracked. In the future, there will be the possibility to track different or multiple shards.
+
+```
+{
+  ...
+  "archive": true,
+  "tracked_shards": [0],
+  ...
+}
+```
+
+Please make sure that the node is stopped while changing the `config.json`.
+
+Once the config has been changed, you can restart the node and the node will start syncing new archival data. In the case where you want the full archival history, you can delete the data dir and start the node from scratch syncing full history or use one of the latest backups containing the data directory snapshot which can be copied under the near home dir (default: ~/.near/data).
+
+All archival data backups can be downloaded from the public S3 bucket, which contains latest daily snapshots:
+
+| Network | URL                                                                                         |
+| ------- | ------------------------------------------------------------------------------------------- |
+| Mainnet | https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/mainnet/archive/data.tar |
+| Testnet | https://near-protocol-public.s3.ca-central-1.amazonaws.com/backups/testnet/archive/data.tar |
+
+---
+
+## Steps to Run an Archival Node
+
+Make sure [`nearup`](https://github.com/near/nearup) is installed. You can install `nearup` by following the instructions at https://github.com/near/nearup.
```

Comment:
We should add instructions on how to run an archival node without nearup as well.

Reply:
Do we also added the tracked shards in the documentation for the archival nodes? Here is seems we only mention the archive: true flag?

Reply:
@janewang feel free to consult with @chefsale on this if you are not sure about how to run an archival node without nearup.

Reply:
@janewang feel free to ping me in chat or even schedule a meeting on running a neard standalone. I would really like to help on this, also provide maybe a guide on how to run it with systemd as well, as this is the best way to do it and that is how we manage all our current nodes on testnet and mainnet.

Reply:
Thanks @chefsale. Would love to connect on neard and systemd. Thanks in advance on this!
@bowenwang1996 Will merge this PR as it stands and add archival instructions with @chefsale's help. Would like to make announcements in validators channels today and link to the docs. Thanks!